test/JDBC/upgrade/latest/schedule

```diff
@@ -313,6 +313,7 @@ sys-objects
 sys-original_login
 sys-partitions
 sys-partitions-dep
+sys-parsename-dep
```

Comment:
you have two sets of test sys-parsename-dev and sys-parsename 
have you verified if both will be running ?

Reply:
YES